examples/jsm/nodes/core/NodeBuilder.js

```diff
@@ -1087,6 +1104,18 @@ class NodeBuilder {
 
 	}
 
+	getPrimitiveType( type ) {
```

Comment:
But we already have `getComponentType` for exactly the same task, no? /ping @sunag

Reply:
I agree! I will test it.